### PR TITLE
Fix pom properties

### PR DESCRIPTION
**Pull request by KCeh**
## Summary

I integrated Gradle Maven publishing plugin into Sentinel, I followed PR for lib template: https://github.com/infinum/android-library-template/pull/14
Then I notice missformatting in POM properties. I am fixing it on lib template so new libs don't have this bug

## Changes

Fix for POM properties

### Type

- [ ] **Feature**: This pull request introduces a new feature.
- [x] **Bug fix**: This pull request fixes a bug.
- [ ] **Refactor**: This pull request refactors existing code.
- [ ] **Documentation**: This pull request updates documentation.
- [ ] **Other**: This pull request makes other changes.

#### Additional information

- [ ] This pull request introduces a **breaking change**.

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [ ] I have updated the documentation to reflect my changes (if applicable).

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -34,8 +34,8 @@ POM_SCM_URL=https://github.com/username/mylibrary/
 POM_SCM_CONNECTION=scm:git:git://github.com/username/mylibrary.git
 POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/username/mylibrary.git
 
-POM_ORGANIZATION_NAME = "Infinum Inc."
-POM_ORGANIZATION_URL = "https://infinum.com"
+POM_ORGANIZATION_NAME=Infinum Inc.
```
[the remark this answers is not in the document]
Comment (KCeh) [reply gradle.properties:37]: This issue didn't pop up with other properties which have spacings - checked on real libs

**Review by AsimRibo — APPROVED** [review]

**Review by JonatanPlesko — APPROVED** [review]
